assets/css/dup/free_text.scss

```diff
@@ -39,8 +39,13 @@
   font-family: Helvetica, sans-serif;
   font-weight: 700;
   text-align: center;
+  transform: translateY(-20px);
```

Comment:
Why are these necessary? I don't think the Helvetica leading issue would affect the positions of these, is there a separate problem?

Reply:
Yeah, the pill itself sat a bit low on the line, so I broke the text element out into a child div and adjusted both the pill and the text's vertical alignment to look correct.